jdaviz/core/tools.py

```diff
@@ -40,7 +40,8 @@ class _BaseZoomHistory:
     # Mixin for custom zoom tools to be able to save their previous zoom state
     # which is then used by the PrevZoom tool
     def save_prev_zoom(self):
-        self.viewer._prev_limits = self.viewer.get_limits()
+        if hasattr(self.viewer, "get_limits"):
+            self.viewer._prev_limits = self.viewer.get_limits()
```

Comment:
So what happens if you press the `previous zoom` button in these viewers? Nothing?

Reply:
For this particular bug, it is the Plot Options histogram, so there is no previous zoom button. It just somehow inherited that method.